index.d.ts

```diff
@@ -93,14 +100,15 @@ declare const pTimeout: {
 	pTimeout(delayedPromise(), 50, () => {
 		return pTimeout(delayedPromise(), 300);
 	});
+	promise.clear(); //Clear timeout
```

Comment:
This is not relevant to the current example, because it makes some of the code moot here. Please remove.